### ISSUE TITLE
Consider adding a todo list

### ISSUE THREAD

**Issue by BenediktHensen** [rwth-acis/LearningPlanVisualization#9]
There could be a general todo list - maybe even an integration with an existing todo list app.

This way, we can get new task items into the application and the user can insert them into their learning plan at a later stage.

Such a todo list can also a quick overview of tasks that are currently not blocked by a previous task since we can filter for achievable tasks.
